File: output/ChatGPT/codenet/Python/C++/s993625131.cpp
#include <iostream>
#include <string>

int main() {
    std::string S;
    std::cin >> S;

    int count = 0;
    int record = 0;
    for (int i = 0; i < S.size(); i++) {
        if (S[i] == 'A' || S[i] == 'C' || S[i] == 'G' || S[i] == 'T') {
            count += 1;
        } else {
            if (count > record) {
                record = count;
            }
            count = 0;
        }
    }
    if (count > record) {
        record = count;
    }

    std::cout << record << std::endl;


    return 0;
}

File: output/StarCoder/codenet/Python/Java/s993625131.java
import java.util.*;
public class s993625131 {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String S = in.next();
        int count = 0;
        int record = 0;
        for (int i = 0; i < S.length(); i++) {
            if (S.charAt(i) == 'A' || S.charAt(i) == 'C' || S.charAt(i) == 'G' || S.charAt(i) == 'T') {
                count++;
            } else {
                if (count > record) {
                    record = count;
                }
                count = 0;
            }
        }
        if (count > record) {
            record = count;
        }
        System.out.println(record);
    }
}